build_packages

```diff
@@ -285,6 +286,48 @@ export KBUILD_BUILD_HOST="${BUILD_HOST:-pony-truck.infra.kinvolk.io}"
 info "Merging board packages now"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" "$@"
 
+info "Merging sysext packages now"
+for sysext in "${EXTRA_SYSEXTS[@]}"; do
+  IFS="|" read SYSEXT_NAME PACKAGE_ATOMS USEFLAGS ARCHES < <(echo "$sysext");
+
+  arch_array=(${ARCHES//,/ })
+  if [[ ! -z "$ARCHES" ]]; then
```

Comment:
No one likes double negatives. :wink: Also, you don't need to quote the LHS in `[[`.

```suggestion
  if [[ -n $ARCHES ]]; then
```

It can even be just `[[ $ARCHES ]]` although my personal preference is to be a little more explicit.